### ISSUE TITLE
[UX] Improve the description text on the block class field

### ISSUE THREAD

**Issue by jenlampton** [backdrop/backdrop-issues#4290]
We had [a forum post](https://forum.backdropcms.org/forum/additional-css-classes-field-configure-block-window#comment-1629) where someone asked if it was the class name that goes in here or the CSS code. Let's update the description text to indicate that the class name goes in here. 

Proposed text: `Separate class names with spaces. Example: <code>wrapper clearfix</code>`

We tried this once before, see https://github.com/backdrop/backdrop-issues/issues/3608

---
PR: https://github.com/backdrop/backdrop/pull/3047

**Comment by stpaultim** [comment]
When looking at this field, I always struggle to remember if I need to include the "." with my class. Should it be `container` or `.container`. I tested this last night, and it turns out that Backdrop can handle both, but I don't know that and I worry about it. 

I would recommend that we provide a sample class name, but that seems wrong given that both versions work. I would like something like: `Separate multiple class names with spaces (preceding . is not necessary).` 

But, this is too wordy and could be just as confusing. Maybe this is better. It gives an example without the ., which I would find helpful. 

`Separate multiple class names with spaces (example: 'container 12-col-wide').`

**Comment by olafgrabienski** [comment]
> Separate multiple class names with spaces (example: class-one class-two).

An example is great. We could even mention real world utility classes provided by core (e.g. from https://github.com/backdrop/backdrop/blob/1.x/core/modules/system/css/system.css):

> Separate multiple class names with spaces. Example: `centered-wrapper clearfix`

**Comment by herbdool** [comment]
Ready, based on @klonos review.

**Comment by olafgrabienski** [comment]
Wondering if we should still add examples to the help text as discussed above):

> Separate multiple class names with spaces. Example: `centered-wrapper clearfix`

**Comment by herbdool** [comment]
I think that's a good idea @olafgrabienski

**Comment by jenlampton** [comment]
PR updated, now both sets of description text read as follows:

Separate class names with spaces. Example: `wrapper clearfix`

I removed both 'multiple' and 'centered-' because the example wrapped to a second line right at the space, thus making it hard to tell that the class names were separated by a space. An alternate solution to this problem would be to increase the width of this modal, something that might be a good idea anyway.

**Comment by olafgrabienski** [comment]
> I removed both 'multiple' and 'centered-' because the example wrapped to a second line right at the space, thus making it hard to tell that the class names were separated by a space.

Good point! It would be nice to have `centered-wrapper`, though, as it is a class with a visible effect (`.centered-wrapper { text-align: center; }`). I've tested to stick with it, and it turns out that the removal of `multiple` already does the job:

<img alt="screen-backdrop-pr-3047-block-classes" src="https://user-images.githubusercontent.com/14188307/84823506-bdeaa380-b01e-11ea-8d36-6f249b75a088.png">

**Comment by jenlampton** [comment]
It isn't enough in Firefox. 
<img width="658" alt="Screen Shot 2020-06-16 at 2 15 53 PM" src="https://user-images.githubusercontent.com/397895/84829054-f321d200-afdb-11ea-8787-364df94f2d6f.png">

I was thinking we might look for another example class that is both short and has a visible effect, or better yet, increase the size of the modal.

**Comment by docwilmot** [comment]
I'm confused here: this is the wording already in core isnt it? What exactly is this issue for?

**Comment by olafgrabienski** [comment]
@docwilmot Wording in core is: `Separate multiple classes with spaces.`

The PR it adds an example: `Separate class names with spaces. Example: <code>wrapper clearfix</code>`

**Comment by stpaultim** [comment]
Works for me. @klonos, can we push this to RTBC?

**Comment by klonos** [comment]
👍 ...yup, let's get this out of the way.

**Comment by BWPanda** [comment]
Thanks @jenlampton, @klonos, @stpaultim, @olafgrabienski & @herbdool! I've merged https://github.com/backdrop/backdrop/pull/3047 into 1.x and 1.17.x.
